### ISSUE TITLE
Backdrivable Kinematically Redundant (6+3)-DOF Hybrid Parallel Robot by Laval University Robotics Laboratory

### ISSUE THREAD

**Issue by hobofan** [hobofan/collected-robotic-arms#13]
Not sure if it makes sense to start adding university projects to this list (vs. more directly acquireable commercial and open source robotic arms), but I found this one interesting.

https://www.youtube.com/watch?v=29pGFE3Xy2M
https://doi.org/10.1109/ICRA.2019.8793772

**Comment by gregsadetsky** [comment]
Yeah, that's an interesting idea -- if there was a field in the data to note that it's a university project, I think that this could work!
